src/scripts/components/Actions.js

```diff
@@ -2,8 +2,59 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+import helpers from '../helpers';
 
 
+/**
```

Comment:
I agree. I spent a bit of time trying to figure out a better way to go about it, played around with gulp a bit, etc. Couldn't figure anything out I felt comfortable with and didn't want that to be what holds up this.

If someone wants to refactor it I'd be quite happy :)